rearrange notification related docs, add new section and info (#448)
worldcoin/world-id-docs: src/pages/mini-apps/notifications/features-and-guidelines.mdx

# Features & Guidelines

## Guidelines

To keep notifications high quality, we enforce the following guidelines:

-   No marketing notifications. These should be purely functional notifications,
-   no notifications that are not related to the mini app,
-   no notifications that are not related to the user.

## Username Substitution

> It's super easy to personalize your notifications, by using the receiving user's username.

Reviewer: " using the receiving user's username." Sounds weird: 
options: "by adding usernames" / "by using user's username"

> Use a special string `${username}` in your notification message, and it'll get automatically substituted by a corresponding username

Reviewer 2: ```suggestion
Use the special string `${username}` in your notification message, and it will substitute in the recipient's username
```

before sending it to the user.

Example message:<br/>
🧑‍🍳 We're cooking something special for you \$\{username\} <br/>
when sent to users, becomes:<br/>
🧑‍🍳 We're cooking something special for you mistico<br/>
🧑‍🍳 We're cooking something special for you tute<br/>
🧑‍🍳 We're cooking something special for you struck<br/>
and so on

## Notification Badges

If a user has pending notifications for your miniapp, and you maintain

> a 7day rolling average open rate of 15% or higher, we'll show a badge on the user's home screen,

Reviewer: 7 day

next to your miniapp's icon.

It's cleared once the miniapp is opened.

## Notification Permission Pause

Given that notifications can be disruptive and reflect negatively on the entire miniapp ecosystem,
we enforce additional rules based on open rates. If your miniapp's notification permissions are enabled,
we evaluate your notification open rates weekly (from May 5th 2025 on).

If your miniapp's 7day rolling average is less than 10%, we pause the permission for a week.

> This is a signal for you to rethink your strategy. After the pause, your permissions will be resumed,

Author: that is true will think of something
